requirements.txt

```diff
@@ -7,6 +7,7 @@ jupyter_telemetry
 oauthlib>=3.0
 pamela
 prometheus_client>=0.0.21
+psutil>=5.6.5
```

Comment:
I'd prefer to avoid adding an unconditional dependency on a compiled extension that doesn't have non-Windows wheels that's only needed on Windows. Perhaps this could be a conditional dependency, only used on Windows?

Reply:
All right, I'll do that.

Reply:
FWIW, psutil is also an optional depency on unix, as it's used by utils.print_ps_info (without which it returns None) which is called by app.JupyterHub.log_status